climakitae/explore/vulnerability.py

```diff
@@ -492,6 +492,8 @@ class CavaParams(param.Parameterized):
     event_duration : tuple, default=(1, "day")
         Duration and unit for event analysis. Format: (duration, unit).
         Unit options: "hour", "day".
+    file_name: str
+        Base name for output files.
```

Comment:
This should specify that it means a filename **without** a file extension, I think this isn’t immediately obvious.